src/renderer/components/Scrubber.tsx

```diff
@@ -0,0 +1,139 @@
+  },
+
+  [`& .${sliderUnstyledClasses.rail}`]: {
+    display: 'block',
+    position: 'absolute',
+    width: '100%',
+    height: '5px',
+    backgroundColor: colors.grey[300],
+  },
+
+  [`& .${sliderUnstyledClasses.track}`]: {
+    display: 'block',
+    position: 'absolute',
+    height: '5px',
+    backgroundColor: `${colors.yellow[500]}CC`,
+  },
+
+  [`& .${sliderUnstyledClasses.thumb}`]: {
+    position: 'absolute',
+    width: '15px',
+    height: '15px',
+    marginLeft: '-5px',
+    marginTop: '-5px',
+    borderRadius: '50%',
+    backgroundColor: colors.yellow[500],
+    opacity: '0',
+    transition: 'opacity 0.2s ease',
+  },
+});
+
+const TimestampContainer = styled('div')({
+  display: 'flex',
+  flex: 1,
+  alignItems: 'flex-start',
+  justifyContent: 'space-between',
+  marginTop: -5,
+});
+
+const Timestamp = styled('div')({
+  fontFamily: '"Roboto Mono", monospace',
+  fontSize: 12,
+});
+
+interface ScrubberProps {
+  totalDuration: number;
+  currentTimeSeconds: number;
+  onScrubberChange: (newTime: number) => void;
+}
+
+const Scrubber = ({
+  totalDuration,
+  currentTimeSeconds,
+  onScrubberChange,
+}: ScrubberProps) => {
+  const [showRemainingTime, setShowRemainingTime] = useState(false);
+
+  const sliderValue = useMemo(() => {
+    // Prevent the slider from going past the end of the video
+    const currentTime =
+      currentTimeSeconds > totalDuration ? totalDuration : currentTimeSeconds;
+
+    // Round to two dp to avoid precision errors
+    const percentageCompleted =
+      Math.round((currentTime / totalDuration + Number.EPSILON) * 10000) / 100;
+
+    return percentageCompleted;
+  }, [currentTimeSeconds, totalDuration]);
+
+  const sliderValueToSeconds = (value: number) =>
+    roundToMs((value / 100) * totalDuration);
+
+  const getNewTime = (newSliderValue: number | number[]) => {
+    // The MUI slider component can return a number or an array of numbers. For this usage
+    // we only get a single number, however we still need to check if the value is an array or
+    // not. If it is an array, we take the first value. This does however have a side effect
+    // of the value being null, in this case, we just return the current timestamp.
+    if (typeof newSliderValue === 'number') {
+      return sliderValueToSeconds(newSliderValue);
+    }
+
+    const value = newSliderValue.shift();
+    return value ? sliderValueToSeconds(value) : currentTimeSeconds;
+  };
+
+  const getRemainingTimestamp = (duration: number, currentTime: number) => {
+    const remainingSeconds = duration - currentTime;
+
+    // Sometimes the actual video playback timer (currentTime) is ever so slightly more then the
+    // transcript total duration. This causes the remaining time to be negative. In this case we
+    // just return 0.
+    if (remainingSeconds < 0) {
+      return `-${secondToTimestampUI(0, false)}`;
+    }
+
+    return `-${secondToTimestampUI(remainingSeconds, false)}`;
```

Comment:
Just make the whole call

`secondToTimestampUI(Math.max(0, remainingSeconds), false)`

Then no need for conditional

Reply:
done.